.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+### Description
+<!-- Please provide a general introduction to the issue/proposal. -->
+
+
+<!--
+If reporting a bug, attach the *entire* traceback from Python.
+If proposing an enhancement/new feature, provide links to related articles, reference examples, etc.
+
+If you are asking a question, please ask on StackOverflow and use the cartopy tag. All cartopy
```

Comment:
What if you are submitting a feature request?

Reply:
I considered that covered by ``If proposing an enhancement/new feature,..``. Is the newline the issue or is the content not quite right?

Reply:
Oh yeah. In that case the (lack of) newline is probably the issue here! I was expecting one paragraph per concept so I completely blanked the second sentence, assuming it to be connected to the first one.